commit-rights.md

```diff
@@ -0,0 +1,72 @@
+Committers Guidelines for community.general
+===========================================
+
+This document is based on the [Ansible committer guidelines](https://github.com/ansible/ansible/blob/b57444af14062ec96e0af75fdfc2098c74fe2d9a/docs/docsite/rst/community/committer_guidelines.rst).
```

Comment:
```suggestion
This document is based on the [Ansible committer guidelines](https://docs.ansible.com/ansible/devel/community/committer_guidelines.html).
```
it has been updated finally:)

Reply:
I intentionally used the link to that commit, since that was the precise version of the document I copied everything from. The document you link to is a large refactoring :) How about:
```suggestion
This document is based on the [Ansible committer guidelines](https://github.com/ansible/ansible/blob/b57444af14062ec96e0af75fdfc2098c74fe2d9a/docs/docsite/rst/community/committer_guidelines.rst) ([latest version](https://docs.ansible.com/ansible/devel/community/committer_guidelines.html)).
```
?

Reply:
ah, clear now, the suggested version also works:)